### PR TITLE
Remove `this.setupRouter` from tests

### PR DESCRIPTION
**Pull request by xg-wang**
This is no longer needed when https://github.com/emberjs/ember.js/pull/19326 lands in the next release (should be 3.25.0, but I could be wrong)

### CODE REVIEW

**Comment by xg-wang** [comment]
The fix is released in 3.25.1, 3.24.2 https://github.com/ember-learn/super-rentals-tutorial/pull/176

File: src/markdown/tutorial/part-2/09-route-params.md [hunk 1]
```diff
@@ -114,43 +114,9 @@ Alright, we have just one more step left here: updating the tests. We can add an
...
-+  hooks.beforeEach(function () {
-+    this.owner.setupRouter();
-+  });
-+
-   test('it renders information about a rental property', async function (assert) {
-```
-
-> Zoey says...
->
-> As its name implies, the `beforeEach` hook runs *once* before each `test` function is executed. This hook is the ideal place to set up anything that might be needed by all test cases in the file. On the other hand, if you need to do any cleanup after your tests, there is an `afterEach` hook!
```
Comment (chancancode) [inline src/markdown/tutorial/part-2/09-route-params.md:144]: Do we need to move this somewhere else? Do we use beforeEach elsewhere?

**Review by chancancode — COMMENTED** [review]
Nice! Can you also rebase to kick CI?

**Comment by xg-wang** [comment]
hmm the test is still failing, I'm looking into the regression

File: src/markdown/tutorial/part-2/09-route-params.md (hunk 1 above)
Comment (xg-wang) [reply src/markdown/tutorial/part-2/09-route-params.md:144]: Good catch! It is indeed taught on this page. I've added it back

**Comment by xg-wang** [comment]
@chancancode this is passing on new releases

**Comment by chancancode** [comment]
🎉

**Pull request merged**